index.d.ts

```diff
@@ -0,0 +1,103 @@
+/// <reference types="node" />
+
+declare module 'cloud-config-client' {
+
+    import http = require('http');
```

Comment:
Let's use `;` everywhere or not use them at all

Reply:
Yes sure, you can adjust as you think it is the best.

Reply:
This one I am not sure where do you mean `;` are still missing. I kind of referenced myself to `mongoose` types.